network-api/networkapi/buyersguide/templates/buyersguide_home.html

```diff
@@ -69,10 +74,10 @@ <h1 class="h1-heading text-center mt-4">Be Smart. Shop Safe.</h1>
             <img
             class="product-thumbnail"
             src="{% cloudinary_url product.cloudinary_image quality=50 fetch_format="auto" crop="fit" width=600 %}"
-            alt="link to {{product.name}}"
+            alt="{% blocktrans with product=product.name %}link to {{product}}{% endblocktrans %}"
             >
             {% else %}
-            <img class="product-thumbnail" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}" alt="link to {{product.name}}">
+            <img class="product-thumbnail" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}" alt="{% blocktrans with product=product.name %}link to {{product}}{% endblocktrans %}">
```

Comment:
```suggestion
            <img class="product-thumbnail" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image }}" alt="{% blocktrans with product_name=product.name %}link to {{product_name}}{% endblocktrans %}">
```
just to avoid confusion when looking at the template while working on python code.